core/src/banking_stage.rs

```diff
@@ -605,8 +606,13 @@ impl BankingStage {
                             );
 
                             match scheduler_controller.run() {
-                                Ok(_) => {}
-                                Err(SchedulerError::DisconnectedRecvChannel(_)) => {}
+                                Ok(_) => info!("Scheduler exiting without error"),
+                                Err(SchedulerError::DisconnectedRecvChannel(_)) => {
+                                    info!("Upstream disconnected, shutting down banking");
+
+                                    shutdown_signal.cancel();
+                                    drop(scheduler_controller);
```

Comment:
nit: maybe helpful to add comment to note the order of shutdown_signal.cancel() and drop(scheduler_controller)?